temba/ussd/models.py

```diff
@@ -31,17 +31,18 @@ class USSDSession(ChannelSession):
     class Meta:
         proxy = True
 
-    def start_session_async(self, flow):
-        flow.start([], [self.contact], start_msg=None, restart_participants=True, session=self)
+    def start_session_async(self, flow, urn, content, date, message_id):
+        from temba.msgs.models import Msg, USSD
+        message = Msg.create_incoming(
```

Comment:
this will call msg.handle() that we don't need, because we want flow.start() to handle it